sklearn/neighbors/tests/test_lof.py

```diff
@@ -76,8 +76,7 @@ def test_lof_performance():
     assert_greater(roc_auc_score(y_test, y_pred), .99)
 
 
-@ignore_warnings(category=DeprecationWarning)
-# contamination changed to 'auto' 0.22
+@pytest.mark.filterwarnings('ignore:default contamination parameter 0.1')
```

Comment:
Should we be giving the version number of scikit-learn where this line should be remove: "# 0.22" at the end of the line?